src/common.js

```diff
@@ -46,7 +46,11 @@ function handleError(error, stdout, stderr) {
   } else if (
     stderr &&
     (stderr.includes("FAILED (remote: not supported in locked device)") ||
-      stderr.includes("FAILED (remote: 'Bootloader is locked.')"))
+      stderr.includes("FAILED (remote: 'Bootloader is locked.')") ||
+      stderr.includes("FAILED (remote: 'not allowed in locked state')") ||
+      stderr.includes(
+        "FAILED (remote: 'Device not unlocked cannot flash or erase')"
```

Comment:
linbreak was intentional?

Reply:
Yep, linter rules. Some day we'll rework the rules, but that day is not today. Thx for review.